template.yaml

```diff
@@ -96,6 +69,8 @@ Outputs:
   CostReportFunctionArn:
     Description: "Cost Report Lambda Function ARN"
     Value: !GetAtt CostReportFunction.Arn
+    Export:
+      Name: "CostReportFunctionArn"
```

Comment:
you should prepend this name with the stack name so that it's unique.  example, `Name: !Sub '${AWS::StackName}-CostReportFunctionArn'`.  in certain cases you may also want to prepend with a region because you can may need to deploy lambdas spread across multiple regions so we sometimes set it to `Name: !Sub '${AWS::Region}-${AWS::StackName}-CostReportFunctionArn'`